init.lua

```diff
@@ -243,11 +262,19 @@ local function register_air_tank_3(name, desc, color, uses, material)
 		inventory_image = "airtanks_airtank_three.png^[colorize:"..color.."^[mask:airtanks_airtank_three.png",
 		wield_image = "airtanks_airtank_three.png^[colorize:"..color.."^[mask:airtanks_airtank_three.png",
 		stack_max = 1,
+		_mcl_armor_element = "torso",
+		_mcl_armor_texture = "airtanks_chestplate_tank_three.png",
 	
 		on_place = function(itemstack, user, pointed_thing)
-			return use_airtank(itemstack, user)
+		   if minetest.get_modpath("mcl_armor") then
+		      mcl_armor.equip_on_use(itemstack, user, pointed_thing)
+		   else
+		      return use_airtank(itemstack, user)
+		   end
 		end,
 
+		on_secondary_use = minetest.get_modpath("mcl_armor") and mcl_armor.equip_on_use,
```

Comment:
`mcl_armor` must be in mod.conf to guarantee that it is loaded before this mod. Internal mcl dependency trees might already do that but it's better to be safe than sorry.

That would also allow you to cache `minetest.get_modpath("mcl_armor")` in a separate variable like `local HAVE_MCL_ARMOR`.

Reply:
done